docs/OBD2-emulator.md

```diff
@@ -0,0 +1,25 @@
+# OBD2 emulator
+
+OBD2 emulator is an query server for endpoint OBDMonitor to get the vehicle responds to the PID query on the CAN bus.
+The OBD2 emulator is implemented according to SAE standard. For more information. You can refer to [OBD-II PIDs](https://en.wikipedia.org/wiki/OBD-II_PIDs).
+
+## How to build OBD2 emulator
+```
+bazel build //endpoint/OBDComp/emulator:obd-emulator
```

Comment:
Would you stick to `bazel` for OBD-II emulator?

Reply:
Actually, I prefer to use gnu make instead.